flexget/plugins/filter/series.py

```diff
@@ -154,6 +155,12 @@ def upgrade(ver, session):
         series_table = table_schema('series', session)
         session.execute(update(series_table, series_table.c.identified_by == None, {'identified_by': 'auto'}))
         ver = 13
+    if ver == 13:
+        # Create default series list
+        series_list_table = table_schema('series_list_lists', session)
+        result = session.execute(series_list_table.insert().values(name=DEFAULT_SERIES_LIST_NAME, added=datetime.now()))
+        table_add_column('series', 'list_id', Integer, session, result.inserted_primary_key[0])
```

Comment:
Not sure about relationships across plugins .. doesn't seem like a good idea. Series list would be a core part of series plugin so it should probably be in the ever so large series plugin ...